lib/core/file.nit

```diff
@@ -663,6 +663,13 @@ class Path
 		return res
 	end
 
+	# Is `self` the path to a directory ?
```

Comment:
Is `self` exist and is the path to a directory?
￼
~~~nit
assert ".".is_dir
assert not "/etc/issue".is_dir
assert not "/should/not/exist".is_dir
~~~